compiler/src/dotty/tools/dotc/core/Flags.scala

```diff
@@ -450,8 +450,12 @@ object Flags {
    *  is completed)
    */
   val AfterLoadFlags: FlagSet = commonFlags(
-    FromStartFlags, AccessFlags, Final, AccessorOrSealed, LazyOrTrait, SelfName, JavaDefined)
-
+    FromStartFlags, AccessFlags, Final, AccessorOrSealed, LazyOrTrait, SelfName, JavaDefined,
+    // We would like to add JavaEnumValue to this set so that we can correctly
+    // detect it in JavaNullInterop. However, JavaEnumValue is not initialized at this
+    // point, so we just make sure that all the "primitive" flags contained in JavaEnumValue
+    // are mentioned here as well.
+    Enum, StableRealizable)
```

Comment:
How do we know that these are indeed never changed after loads? Assuming a flag as an after load flag that can nevertheless still be changed later produces a kind of race that we want to avoid. This will be checked once #7427 is in.

Reply:
Removed.